spec/reek/smell_detectors/utility_function_spec.rb

```diff
@@ -177,6 +177,53 @@ def bravo(charlie)
     end
   end
 
+  context 'when examining class_methods block' do
+    it 'does not report methods defined in class_methods block' do
+      src = <<-RUBY
+        module Alpha
+          extend ActiveSupport::Concern
+
+          class_methods do
+            def bravo(arg1, arg2)
+              return 1 if arg1.something?
+              return 2 if arg2.something_else?
+
+              3
+            end
+          end
+        end
+      RUBY
+
+      expect(src).not_to reek_of(:UtilityFunction, context: 'Alpha#bravo')
+    end
+
+    it 'reports methods defined outside the class_methods block' do
+      src = <<-RUBY
+        module Alpha
+          extend ActiveSupport::Concern
+
+          class_methods do
+            def bravo(arg1, arg2)
+              return 1 if arg1.something?
+              return 2 if arg2.something_else?
+
+              3
+            end
+          end
+
+          def charlie(arg1, arg2)
+            return 4 if arg1.something?
+            return 5 if arg2.something_else?
+
+            6
+          end
+        end
+      RUBY
+
+      expect(src).not_to reek_of(:UtilityFunction, context: 'Alpha#charlie')
```

Comment:
@mateusluizfb Thanks for this PR!

I have a question about this line. `charlie` is defined _outside_ of the `class_methods do` block.
So I would expect it to raise a `UtilityFunction` warning. 

Can you help me understand why this expectation is a `.not_to reek_of`? Thanks.